web/src/settings_config.ts

```diff
@@ -114,6 +114,13 @@ export const web_stream_unreads_count_display_policy_values = {
     },
 };
 
+// <li> classes, The default order of left_sidebar possible home views.
+export const possible_home_views_default_order = [
```

Comment:
The information here is basically a duplicate of the `web_home_view_values`, and `top_left_xxx` is very much a front-end thing. I don't like how this smells sitting amongst legitimate settings options.

Reply:
Hmm, maybe we can put `possible_home_views_default_order` elements inside `web_home_view_values` ?

Reply:
or leave it as it is and just move `possible_home_views_default_order` to `left_sidebar_navigation_area`

Reply:
@sahil839, do you have thoughts about this?

Reply:
One way what we do for options of a setting dropdown, is have a `order` field and then use that to sort. So, maybe `web_home_view_values` objects can have a `order` field along with `code` and `description`.

Reply:
@sahil839 do we really need that `order` field ?  the objects inside `web_home_view_values` are already ordered in the same order as the default visible order

Reply:
Ohh sorry, I misunderstood the question. Putting the values in `possible_home_views_default_order` in `web_home_view_values` seems fine.

Reply:
Am I right in the point in which the order of  `web_home_view_values` keys is not guaranteed to be always as the same order as defined

Reply:
Yes, it can be changed if we decide to change the order of options in the setting dropdown. Maybe a comment can be helpful if we want to use the same object to make sure person changing the order makes sure to not break keyboard accessibility.